manage-data/lifecycle/index-lifecycle-management/tutorial-customize-built-in-policies.md

```diff
@@ -19,8 +21,16 @@ products:
 * `traces@lifecycle`
 * `traces-otel@lifecycle`
 
-{{agent}} uses these policies to manage backing indices for its data streams. This tutorial shows you how to use {{kib}}’s **Index Lifecycle Policies** to customize these policies based on your application’s performance, resilience, and retention requirements.
+This tutorial gives the example of customizing how ingested logging data is managed. Rather than use the default lifecycle settings from the built-in `logs@lifecycle` {{ilm-init}} policy, you can use the **Index Lifecycle Policies** feature in {{kib}} to tailor a new policy based on your application’s specific performance, resilience, and retention requirements. This involves three main steps:
```

Comment:
Minor suggestion to simplify:

```suggestion
This tutorial covers customizing the way ingested logging data is managed. Rather than use the default lifecycle settings from the built-in `logs@lifecycle` {{ilm-init}} policy, you can use the **Index Lifecycle Policies** feature in {{kib}} to tailor a new policy based on your application’s specific performance, resilience, and retention requirements. This involves three main steps:
```

Reply:
nice catch!